pkg/exchange/bitget/exchange.go

```diff
@@ -368,3 +370,59 @@ func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) erro
 	// TODO implement me
 	panic("implement me")
 }
+
+// QueryTrades queries fill trades. The trade of the response is in descending order. The time-based query are typically
+// using (`CTime`) as the search criteria.
+// If you need to retrieve all data, please utilize the function pkg/exchange/batch.TradeBatchQuery.
+//
+// ** StartTime is inclusive, EndTime is exclusive. If you use the EndTime, the StartTime is required. **
+// ** StartTime and EndTime cannot exceed 90 days. **
+func (e *Exchange) QueryTrades(ctx context.Context, symbol string, options *types.TradeQueryOptions) (trades []types.Trade, err error) {
+	if options.LastTradeID != 0 {
+		log.Warn("!!!BITGET EXCHANGE API NOTICE!!! The trade of response is in descending order, so the last trade id not supported.")
+	}
+
+	req := e.v2Client.NewGetTradeFillsRequest()
+	req.Symbol(symbol)
+
+	if options.StartTime != nil {
+		req.StartTime(options.StartTime.UnixMilli())
+	}
+	if options.EndTime != nil {
+		req.EndTime(options.EndTime.UnixMilli())
+		if options.StartTime == nil {
+			return nil, errors.New("start time is required for query trades if you take end time")
+		}
+	}
+
+	limit := options.Limit
+	if limit > queryOrdersLimit || limit <= 0 {
```

Comment:
Should be `queryTradesLimit`?

Reply:
Oh, both the trade and order have a limit of 100, so I am using it. I will rename it to `queryLimit`.